vkubelet/node.go

```diff
@@ -189,6 +188,12 @@ func (n *Node) Run(ctx context.Context) error {
 	if n.disableLease {
 		log.G(ctx).Info("Node leases not supported, falling back to only node status updates")
 	}
+	if n.leases == nil {
+		// now that everything is initialized, just set disableLease to true since
+		// there is no lease client set.
+		// this just makes things easier to read later instead of having a nil check
+		n.disableLease = true
+	}
```

Comment:
Can this `if` block move up to become an `if-else` block which is started in line 171?

Reply:
I didn't want to log the message about falling back if no node lease client was even supplied.

Reply:
Ok, I have a better solution, pushed now.